docs/user/customization.md

```diff
@@ -12,16 +12,162 @@ The most simple customization is exposed by the installer as an interactive seri
+The `install-config.yaml` generated by the installer will not have all of the available fields populated, so they may need to be manually added if they are needed.
+
+The following `install-config.yaml` properties are available:
+
+* `apiVersion` (required string): The API version for the `install-config.yaml` content.
+    The current version (as described in this documentation) is `v1`.
+    The installer may also support older API versions.
+* `additionalTrustBundle` (optional string): a PEM-encoded X.509 certificate bundle that will be added to the nodes' trusted certificate store.
+* `baseDomain` (required string): The base domain to which the cluster should belong.
+* `controlPlane` (optional [machine-pool](#machine-pools)): The configuration for the machines that comprise the control plane.
+* `compute` (optional array of [machine-pools](#machine-pools)): The configuration for the machines that comprise the compute nodes.
+* `imageContentSources` (optional array of objects): Sources and repositories for the release-image content.
+    Each entry in the array is an object with the following properties:
+    * `source` (required string): The repository that users refer to, e.g. in image pull specifications.
+    * `mirrors` (optional array of strings): One or more repositories that may also contain the same images.
+* `metadata` (required object): Kubernetes resource ObjectMeta, from which only the `name` parameter is consumed.
+    * `name` (required string): The name of the cluster.
+        DNS records for the cluster are all subdomains of `{{.metadata.name}}.{{.baseDomain}}`.
+* `networking` (optional object): The configuration for the pod network provider in the cluster.
+    * `clusterNetwork` (optional array of objects): The IP address pool for pods.
+        The default is 10.128.0.0/14 with a host prefix of /23.
+        * `cidr` (required [IP network](#ip-networks)): The IP block address pool.
+        * `hostPrefix` (required integer): The prefix size to allocate to each node from the CIDR.
+        For example, 24 would allocate 2^8=256 adresses to each node.
+    * `machineCIDR` (optional [IP network](#ip-networks)): The IP address pool for machines.
+        The default is 10.0.0.0/16 for all platforms other than libvirt.
+        For libvirt, the default is 192.168.126.0/24.
+    * `networkType` (optional string): The type of network to install.
+        The default is [OpenShiftSDN][openshift-sdn].
+    * `serviceNetwork` (optional array of [IP networks](#ip-networks)): The IP address pool for services.
+        The default is 172.30.0.0/16.
+* `platform` (required object): The configuration for the specific platform upon which to perform the installation.
```

Comment:
gcp, none, baremetal ?

Reply:
> gcp, none, baremetal ?

Can we grow those in follow-up work?  We weren't linking to them before this PR, and I'm not all that familiar with whether parameters are required or not.  Seems like it would be easier to review if they had platform-specific PRs.